constellations/gemini-admin/src/location/locationEdit.js

```diff
@@ -116,35 +117,75 @@ export class LocationEditPage extends React.Component {
             <div id="view-location-edit">
                 {modalDiv}
                 <h2>{title}</h2>
-                <h4>Location ID</h4>
-                <input
+                <InputText
+                    description="Location ID"
+                    required={true}
                     value={this.state.inputlocationId}
-                    onChange={(e) => this.handleChange(e, "inputlocationId")}
+                    onChangeCallback={(e) =>
+                        this.handleChange(e, "inputlocationId")
+                    }
+                    validators={[
+                        {
+                            validate: (text) => text != "",
+                            message: "You must input a location ID",
+                        },
+                    ]}
                 />
-                <h4>Street</h4>
-                <input
+                <InputText
+                    description="Street  (E.g. 1234 Gains Rd, 5432 Abc Drive W)"
```

Comment:
Should be:
label="Street"
description = "Street Number and Street (e.g. 1234 Gains Rd, 5432 Victory Dr)"